cogs/storage.py

```diff
@@ -0,0 +1,45 @@
+import pickle
+
+from dotenv import load_dotenv
+from discord.ext import commands
+
+PICKLE_FILEPATH = './data.pickle'
+
+class DataStore:
+    def __init__(self):
+        self.calendar_cache = {}
+        self.birthday_cache = {}
+        
+class Storage(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.data : DataStore = self.load_data()
```

Comment:
```suggestion
        self.data: DataStore = self.load_data()
```
Would this read better as? If `DataStore` is a data type